client/src/templates/Challenges/video/Show.js

```diff
@@ -222,35 +222,35 @@ export class Project extends Component {
                 <PrismFormatted text={text} />
                 <Spacer />
                 <ObserveKeys>
-                  <fieldset className='video-quiz-options'>
+                  <div className='video-quiz-options'>
                     {answers.map((option, index) => (
                       // answers are static and have no natural id property, so
                       // index should be fine as a key:
-                      <label className='video-quiz-option-label' key={index}>
-                        <span
+                      <label
+                        className={`video-quiz-option-label ${
+                          this.state.selectedOption === index
+                            ? 'outlined'
```

Comment:
There's no css that hooks onto this yet (and I didn't name it well), but this approach seemed like an easy way to highlight the whole box without adding a tabindex.

Reply:
This was a good way to help visualise the selection.